_data/docs.yml

```diff
@@ -35,6 +35,28 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  cugraph:
+    name: cuGraph
+    path: cugraph
+    desc: 'cuGraph is a GPU accelerated graph analytics library, with functionality like NetworkX, which is seamlessly integrated into the RAPIDS data science platform. cuGraph supports GNNs with PyG, DGL packages, cugraph-service for analytics on a remote graph, and WHOLEGRAPH for memory management.'
```

Comment:
Non-blocking: This needs to be updated, the GNN functionality has been split out.

Reply:
I'll address this in a follow-up... it does need to be updated (to remove DGL and to mention the specific library names for searchability), but not because the GNN functionality has been split out.

The whole family of `cugraph` packages builds its docs from one repo: https://github.com/rapidsai/cugraph-docs

Instead of uploading from individual projects' CI like the rest of RAPIDS. That is also why you don't see a separate entry for `nx-cugraph` in https://github.com/rapidsai/docs/blob/main/_data/docs.yml, I think.

I'll put up a follow-up with a proposal we can discuss.

cc @acostadon

Reply:
Put up a proposal at https://github.com/rapidsai/docs/pull/671